core/primitives/src/network.rs

```diff
@@ -12,7 +12,7 @@ use crate::types::{AccountId, EpochId};
 
 /// Peer id is the public key.
 #[derive(
-    BorshSerialize, BorshDeserialize, Clone, Eq, PartialOrd, Ord, Serialize, Deserialize, Hash,
+    Copy, BorshSerialize, BorshDeserialize, Clone, Eq, PartialOrd, Ord, Serialize, Deserialize, Hash,
```

Comment:
Why do we want to make this type copiable?

Reply:
Ok, I  changed the code and removed Copy trait.